test/fixtures/state-saved-v1-6.js

```diff
@@ -750,6 +750,9 @@ export const savedStateWIthNonValidFilters = {
           brush: {
             size: 1,
             enabled: false
+          },
+          geocoder: {
```

Comment:
the saved state should not be changed, the merged config should have geocoder set to `{enablced: false}`